src/constitutive/TACSBladeStiffenedShellConstitutive.h

```diff
@@ -142,13 +154,82 @@ class TACSBladeStiffenedShellConstitutive : public TACSShellConstitutive {
       int _stiffenerHeightNum, TacsScalar _stiffenerThick,
       int _stiffenerThickNum, int _numStiffenerPlies,
       TacsScalar _stiffenerPlyAngles[], TacsScalar _stiffenerPlyFracs[],
-      int _stiffenerPlyFracNums[], TacsScalar _flangeFraction = 1.0);
+      int _stiffenerPlyFracNums[], TacsScalar _flangeFraction = 1.0,
+      bool _includePanelMaterialFailure = true,
+      bool _includeStiffenerMaterialFailure = true,
+      bool _includeGlobalBuckling = true, bool _includeLocalBuckling = true,
+      bool _includeStiffenerColumnBuckling = true,
+      bool _includeStiffenerCrippling = true);
```

Comment:
The function handle for this class is starting to get really long. Is there a way we can condense some of this?
Maybe be converting each of these individual flags to a single binary composite flag, similar to how we handle TACS_OUTPUT (see [here](https://github.com/smdogroup/tacs/blob/master/examples/crm/crm.py#L199-L207)).

Reply:
I could just remove these arguments from the constructor and have the setter methods as the only way to enable and disable different failure modes, same as is currently done for the DV bounds

Reply:
That'd work too